app/screens/channel/channel.js

```diff
@@ -215,7 +218,9 @@ class Channel extends PureComponent {
                         ref={this.attachPostTextbox}
                         navigator={navigator}
                     />
+                    <ChannelLoader theme={theme}/>
                 </KeyboardLayout>
+                {LocalConfig.EnableMobileClientUpgrade && <ClientUpgradeListener navigator={navigator}/>}
```

Comment:
Why isn't this turn on by default? and why is this a local config instead of a server config?